src/pyggdrasil/tree_inference/_simulate.py

```diff
@@ -251,8 +251,142 @@ def attach_cells_to_tree(
     Returns:
         binary matrix of shape ``(n_cells, n_sites)``,
           where ``n_sites`` is determined from the ``tree``
+          NOTE: does not trunicate the last row as shown in the SCITE paper
```

Comment:
I decided to output the full mutation matrix - in difference to what is displayed in the SCITE paper in Eqn. 1, where the last row corresponding to the root is truncated. As it is just a vector of all ones. As the root is the parent of all nodes. Is there any good reason to truncate the last row - issued if I don't?

Reply:
I don't think there's any reason to truncate it.

However, it may be good to keep the convention that the root node is the 0th one. (Or, if you prefer it to be the last one, it'd be good to very clearly specify this convention).

Reply:
(Currently cell attachment vector seems to point out that the 0th node is the root.)